sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -26,6 +52,30 @@ public enum ServiceVersion
             V7_2_Preview = 1,
         }
     }
+    public partial class KeyVaultBackupRestoreClient
```

Comment:
Considering we don't use KeyVault prefix in other clients maybe VaultBackupClient?

Reply:
I think it is more concise. We do use the full prefix also in RBAC. 

I believe the reason we don't use the full `KeyVault` prefix elsewhere is that that no other clients apply to the full KeyVault context. I think `KeyVault` may be more consistent with what someone would expect the name to be, but I don't feel strongly about it.

Reply:
>but I don't feel strongly about it

Me neither